docs/userman/gui/binding.md

```diff
@@ -7,18 +7,37 @@ Consider the following application:
 
 <a name="first-example"></a>
 
-```python linenums="1"
-from taipy import Gui
+=== "Python"
+    ```python linenums="1"
+    import taipy.gui.builder as tgb
+    from taipy import Gui
+
+    x = 1234
+    with tgb.Page() as page:
+        tgb.html("h1", "Hello Taipy")
```

Comment:
Why do we need html here? tgb.text(..., mode="md") is not sufficient?

Reply:
Not only for this line

Reply:
Because there's no need for a text element. That's raw HTML anyways.